elvers/rules/trinity/environment.yml

```diff
@@ -3,4 +3,4 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - trinity
+  - trinity=2.8.4
```

Comment:
When testing this environment.yml, it raises an error about version 2.8.4 not being found. But that's the latest version as listed on trinity's anaconda cloud page: https://anaconda.org/bioconda/trinity

Reply:
Also see #135 which lists several other issues

Reply:
were you testing on mac? mac version is stuck back at  `date.2011_11_26`

Reply:
Bingo - that was the problem. Running this on a vagrant machine (ubuntu 18.04) solved the version not found issue.

Reply:
(It's like you've gone through this whole process before... ;)